### ISSUE TITLE
Problem on mobile devices

### ISSUE THREAD

**Issue by ghost** [braginteractive/MDLWP#22]
If I search long words or a long phrase, the entire page moves to the left.

![screenshot 06-23pm mar 20 2016](https://cloud.githubusercontent.com/assets/17053051/13905765/4e9ad722-eec9-11e5-8251-248503f0cf09.png)

**Comment by braginteractive** [comment]
Will need to look into this. Have you found a fix?

**Comment by ghost** [comment]
No, I haven't. But I've also posted this issue in the Google mdl page.
